samples/sampler/stories/qa/Layout.js

```diff
@@ -0,0 +1,136 @@
+import {boolean, select} from '@enact/storybook-utils/addons/controls';
+import Layout, {Cell, Column, Row} from '@enact/ui/Layout';
+
+export default {
+	title: 'Sandstone/Layout',
```

Comment:
I'm not sure about adding this to sandstone qa-sampler. 🤔
How about adding `grow` prop to the existing enact ui/Layout sampler?

Reply:
I thought sampler and qa-sampler had different purposes.
The sampler had a showcase for UI components, and the qa-sampler had various test cases for UI components. But Enact doesn't have a qa-sampler, so I put this code in Sandstone's qa-sampler. But I will if you want to place this code into Enact's sampler.

Reply:
If we could cover the test with Screenshot tests, I think that is enough.
Let's not add this and see if you can add the `grow` prop to the default Layout sampler from enact sampler.